pcweb/pages/demos_on_landing_page/dashboard/dashboard.py

```diff
@@ -70,7 +70,7 @@ def recent_sale_item(first_name, last_name, sale_amount):
         ),
         rx.spacer(),
         rx.text(formatted_sale_amount, color="#FFFFFF", line_height="1", font_size="16px", align="right"),
```

Comment:
In general, can we use the builtin radix `size` prop everywhere instead of `px`?